docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -437,3 +445,38 @@ See [Configure Grafana]({{< relref "../../../configure-grafana#authazuread" >}})
 # prevents the sync of org roles from AzureAD
 skip_org_role_sync = true
 ```
+
+## Configuration options
+
+The following table outlines the various generic Azure AD/Entra ID configuration options. You can apply these options as environment variables, similar to any other configuration within Grafana.
```

Comment:
nit: maybe add a link to https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables ?

Reply:
Hey @mgyongyosi sorry for extra comment - I am working on another issue where it mentions it is unclear how to avoid any errors when the JMESPath expression includes a colon. As we don't have one single place talking about this - what do you think about adding something around the lines of:

```
Note: If the configuration option requires a [JMESPath](http://jmespath.org/examples.html) expression that includes a colon (:), enclose the entire expression in quotes to prevent parsing errors. For example:

role_attribute_path: "role:view"

```

Reply:
I would prefer to have a separate page for JMESPath examples, instead of adding the same/similar example to all of the pages, wdyt?

Reply:
> I would prefer to have a separate page for JMESPath examples, instead of adding the same/similar example to all of the pages, wdyt?

I agree, let's review together what would be the best way to structure those.

Reply:
FYI bunch of other places where I had to add https://github.com/grafana/grafana/pull/96237/files (not suggesting to merge mine, as we talked above, let's see if we can extract this info to some helper content)